apps/site/src/app/(home)/sections/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,218 @@
+"use client";
+
+import bg_map from "@/assets/images/maps/countdown.svg";
```

Comment:
chore: reorder imports (from top to bottom):

- library (like next.js)
- our own components, functions, etc.
- assets (like svgs)
- stylesheets (like `Countdown.module.scss`)